### DISCUSSION TITLE
qBittorrent-nox android progress

### DISCUSSION THREAD

**Discussion by aicynide** [userdocs/qbittorrent-nox-static#143] [Ideas]
Pull request to add qbittorrent-nox to the termux-packages repo https://github.com/termux/termux-packages/pull/19991

Termux package repo build script (found in PR) - https://github.com/termux/termux-packages/pull/19991/files#diff-7703ec51d10bc01ee89d7da17ea9290f0c345365960614c5195d04fb544b397b

Github CI successfully built qbittorrent-nox https://github.com/termux/termux-packages/actions/runs/8878010622/job/24372772631?pr=19991

Binaries I got from the above github CI for termux aarch64,x86_64,x86 and armv7 - https://github.com/aicynide/flood/releases/download/v4.7.0-1/qbittorrent-nox-binaries-termux.zip
(Note: you need to install libtorrent-rasterbar & qt5-qtbase , libicu, openssl before running the binaries. install by running pkg update && pkg install libicu libtorrent-rasterbar openssl qt5-qtbase in termux )

**Comment by userdocs** [comment]
Please. These are not issues. Use the discussion feature.

**Reply by aicynide** [reply]
What is discussion?
